test/WebSites/AddServicesWebSite/Startup.cs

```diff
@@ -9,6 +9,8 @@ public class Startup
     {
         public void Configure(IApplicationBuilder app)
         {
+            app.UseIISPlatformHandler();
+            
```

Comment:
sorry I didn't understand what you mean. Should I remove the white line? But there are whitelines also in the files in each app.usexxx